### PR TITLE
Update captions banner width and height to increase readability

### PR DESCRIPTION
**Pull request by carocao-msft**
# What
Update captions banner width and height to increase readability

# Why
https://skype.visualstudio.com/SPOOL/_workitems/edit/3245322
https://skype.visualstudio.com/SPOOL/_workitems/edit/3245318

# How Tested
Tested with samples 

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: change-beta/@azure-communication-react-df2fc150-ffbd-4e1a-bb90-ad1ea5304bf2.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```
Comment (dmceachernmsft) [inline change-beta/@azure-communication-react-df2fc150-ffbd-4e1a-bb90-ad1ea5304bf2.json:2]: If this is for Captions GA is this pre-release?

File: packages/react-components/src/components/styles/Captions.style.ts [hunk 2]
```diff
@@ -65,9 +65,13 @@ export const captionsBannerClassName = (formFactor: 'default' | 'compact'): ISta
 /**
  * @private
  */
-export const loadingBannerClassName = mergeStyles({
-  height: _pxToRem(100)
-});
+export const loadingBannerClassName = (formFactor: 'default' | 'compact'): IStackStyles => {
```
Comment (dmceachernmsft) [inline packages/react-components/src/components/styles/Captions.style.ts:68]: Can we change the naming for these styles? I don't agree that they are a classname, we should be calling this `loadingBannerStyles` there are a lot of other classes like this.

Comment (edwardlee-msft) [reply packages/react-components/src/components/styles/Captions.style.ts:68]: I agree with this as well.

**Review by edwardlee-msft — APPROVED** [review]

File: change-beta/@azure-communication-react-df2fc150-ffbd-4e1a-bb90-ad1ea5304bf2.json (hunk 1 above)
Comment (carocao-msft) [reply change-beta/@azure-communication-react-df2fc150-ffbd-4e1a-bb90-ad1ea5304bf2.json:2]: It's feature related changes, I think it should be prerelease?

**Pull request merged**